### PR TITLE
add log-json support

### PR DESCRIPTION
**Pull request by ShakoHo**
add log to json file arg support @askeing r?

### CODE REVIEW

File: b2g_util/get_crashreports.py [hunk 1]
```diff
@@ -59,12 +60,20 @@ def get_crashreports(self, serial=None):
...
             print('The links of Submitted Crash Reports:')
             for line in submitted.split('\n'):
                 submmited_id = re.sub(r'\.txt\s*$', '', re.sub(r'^.+bp-', '', line))
                 submitted_url = 'https://crash-stats.mozilla.com/report/index/{}'.format(submmited_id)
+                submitted_url_list.append(submitted_url)
                 print(submitted_url)
 
+        if self.args.log_json is not None:
+            with open(self.args.log_json, 'w') as outfile:
+                result = {'PendingCrashReportsList': pending, 'SubmittedCrashReportsList': submitted,
```
Comment (askeing) [inline b2g_util/get_crashreports.py]: `pending` and `submitted` are the stdout of adb shell command, not list object.
I will suggest to remove them from result object.

Comment (ShakoHo) [inline b2g_util/get_crashreports.py]: Yes, I know. I expect people who use log-json args will have to handle the stdout by themself, ex: jsonoutput['PendingCrashReportsList'].split('\n')

Maybe I should rename the keyname from PendingCrashReportsList to PendingCrashReportsStdout

Comment (askeing) [inline b2g_util/get_crashreports.py]: agree :)

**Comment by askeing** [comment]
fixed by https://github.com/askeing/b2g-util-python/commit/a03c2d4d5abc025024108e0e0f3b331e98b8e222

**Pull request closed without merging**